scripts/templates/create-page/EmptyStyles.mustache

```diff
@@ -0,0 +1,23 @@
+import { IStyle, getGlobalClassNames } from 'office-ui-fabric-react';
+import { I{{pageName}}PageStyleProps, I{{pageName}}PageStyles } from './{{pageName}}Page.types';
+
+const GlobalClassNames: { [key in keyof I{{pageName}}PageStyles]: string } = {
```

Comment:
Unless you know of a specific reason that we need the pages to support global class names, I'd rather avoid encouraging people to add more of them.

Reply:
Fair, removed from the template.